purchase_work_acceptance_evaluation/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Work Acceptance Evaluation",
+    "version": "13.0.1.0.0",
+    "category": "Purchase Management",
+    "author": "Ecosoft, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/OCA/purchase-workflow",
+    "depends": ["purchase_work_acceptance"],
+    "data": [
+        "data/evaluation_data.xml",
+        "security/ir.model.access.csv",
+        "security/security.xml",
+        "views/res_config_settings_views.xml",
+        "views/work_acceptance_evaluation_views.xml",
+        "views/work_acceptance_views.xml",
+    ],
+    "maintainer": ["kranokporn"],
```

Comment:
```suggestion
    "maintainers": ["kranokporn"],
```
https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L13